packages/core/cypress/global.js

```diff
@@ -13,3 +13,11 @@ export const options = {
     }
   },
 }
+
+// TODO: Bring back these rules when the components with missing/wrong role is found.
```

Comment:
Create an Issue about it, in order for us to remember to add the role on the components.

Reply:
The [FS-1312](https://vtex-dev.atlassian.net/browse/FS-1312?atlOrigin=eyJpIjoiMTg0OTFmY2UyNGIyNGVhYThkMTA5YTQ5Y2ZhNzZkNDUiLCJwIjoiaiJ9) task should track this.